src/frequenz/quantities/_reactive_power.py

```diff
@@ -0,0 +1,213 @@
+    def from_kilo_volt_amperes_reactive(cls, kvars: float) -> Self:
+        """Initialize a new reactive power quantity.
+
+        Args:
+            kvars: The reactive power in kilovolt-amperes reactive (kVAR).
+
+        Returns:
+            A new reactive power quantity.
+        """
+        return cls._new(kvars, exponent=3)
+
+    @classmethod
+    def from_mega_volt_amperes_reactive(cls, mvars: float) -> Self:
+        """Initialize a new reactive power quantity.
+
+        Args:
+            mvars: The reactive power in megavolt-amperes reactive (MVAR).
+
+        Returns:
+            A new reactive power quantity.
+        """
+        return cls._new(mvars, exponent=6)
+
+    def as_volt_amperes_reactive(self) -> float:
+        """Return the reactive power in volt-amperes reactive (VAR).
+
+        Returns:
+            The reactive power in volt-amperes reactive (VAR).
+        """
+        return self._base_value
+
+    def as_milli_volt_amperes_reactive(self) -> float:
+        """Return the reactive power in millivolt-amperes reactive (mVAR).
+
+        Returns:
+            The reactive power in millivolt-amperes reactive (mVAR).
+        """
+        return self._base_value * 1e3
+
+    def as_kilo_volt_amperes_reactive(self) -> float:
+        """Return the reactive power in kilovolt-amperes reactive (kVAR).
+
+        Returns:
+            The reactive power in kilovolt-amperes reactive (kVAR).
+        """
+        return self._base_value / 1e3
+
+    def as_mega_volt_amperes_reactive(self) -> float:
+        """Return the reactive power in megavolt-amperes reactive (MVAR).
+
+        Returns:
+            The reactive power in megavolt-amperes reactive (MVAR).
+        """
+        return self._base_value / 1e6
+
+    # We need the ignore here because otherwise mypy will give this error:
+    # > Overloaded operator methods can't have wider argument types in overrides
+    # The problem seems to be when the other type implements an **incompatible**
+    # __rmul__ method, which is not the case here, so we should be safe.
+    # Please see this example:
+    # https://github.com/python/mypy/blob/c26f1297d4f19d2d1124a30efc97caebb8c28616/test-data/unit/check-overloading.test#L4738C1-L4769C55
+    # And a discussion in a mypy issue here:
+    # https://github.com/python/mypy/issues/4985#issuecomment-389692396
+    @overload  # type: ignore[override]
+    def __mul__(self, scalar: float, /) -> Self:
+        """Scale this power by a scalar.
+
+        Args:
+            scalar: The scalar by which to scale this power.
+
+        Returns:
+            The scaled power.
+        """
+
+    @overload
+    def __mul__(self, percent: Percentage, /) -> Self:
+        """Scale this power by a percentage.
+
+        Args:
+            percent: The percentage by which to scale this power.
+
+        Returns:
+            The scaled power.
+        """
+
+    @overload
+    def __mul__(self, other: timedelta, /) -> Energy:
```

Comment:
We should probably remove the `Energy` interactions for `ReactivePower` and `ApparentPower`. If necessary, they can be added later.